homeassistant/components/ecobee/switch.py

```diff
@@ -88,3 +96,41 @@ async def async_turn_off(self, **kwargs: Any) -> None:
             self.data.ecobee.set_ventilator_timer, self.thermostat_index, False
         )
         self.update_without_throttle = True
+
+
+class EcobeeSwitchAuxHeatOnly(EcobeeBaseEntity, SwitchEntity):
+    """Representation of a aux_heat_only ecobee switch."""
+
+    _attr_has_entity_name = True
+    _attr_name = "Aux Heat Only"
+
+    def __init__(
+        self,
+        data: EcobeeData,
+        thermostat_index: int,
+    ) -> None:
+        """Initialize ecobee ventilator platform."""
+        super().__init__(data, thermostat_index)
+        self._attr_unique_id = f"{self.base_unique_id}_aux_heat_only"
+
+        self._last_hvac_mode_before_aux_heat = HASS_TO_ECOBEE_HVAC.get(
+            HVACMode.HEAT_COOL
+        )
+
+    def turn_on(self, **kwargs: Any) -> None:
+        """Set the hvacMode to auxHeatOnly."""
+        _LOGGER.debug("Setting HVAC mode to auxHeatOnly to turn on aux heat")
```

Comment:
The fact these services are called, is already part of the debug log.

To me, this seems a bit duplicate.

```suggestion
```